mackup/applications/boxer.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Boxer
+
+[configuration_files]
+Library/Preferences/net.washboardabs.boxer.plist
+Applications/DOS Games
```

Comment:
> Is there really an `~/Applications` folder being created?

Yes.

> Do we really want to sync whole games? It might mean a lot of big files no?

These are almost never going to exceed maybe 15 MB at the very most for each game, so it seemed like it could be useful to include.